api/app/Models/User.php

```diff
@@ -32,6 +33,14 @@ class User extends IndexedModel implements AuthenticatableContract
      */
     protected $token_ttl = 1440;
 
+    protected $authMethod;
+
+    protected $remember_token;
+
+    protected $hidden = [
+        'remember_token'
```

Comment:
This is not persisted to the database right? We're pretty much highjacking the remember token interface to store jwt tokens?